### PR TITLE
Update Iceburg SCBs to adjudicate comments made by JCDC during CB publication review

### LINKED ISSUE #1291: Update Iceburg SCBs to adjudicate comments made by JCDC
# 💡 Summary #

Make the appropriate grammatical and consistency updates to the Iceberg SCBs to adjudicate the comments left by the JCDC review team.

## Motivation and context ##

This work is needed to address the comments left by the JCDC in the consolidated change document to sync the markdowns with the change document going through CB publication review.

## Implementation notes ##

Linked to issue #1181 

## Acceptance criteria ##

- [ ] All policies that currently do not have a MITRE ATT&CK mapping will still have a section for possible mapping with "none" listed at the moment to stay consistent with the GWS baselines
- [ ] All other grammatical comments adjudicated

### PR DESCRIPTION
**Pull request by ahuynhMITRE**
## 🗣 Description ##

PR to sync the grammatical and consistency of the language found in the Iceberg SCBs to adjudicate the comments left by the JCDC review team.

Closes #1291 

## 💭 Motivation and context ##

This work is needed to address the comments left by the JCDC in the consolidated change document to sync the markdowns with the change document going through CB publication review.

## 🧪 Testing ##

N/A

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] PR targets the correct parent branch (e.g., main or release-name) for merge.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] Changes are sized such that they do not touch excessive number of files.
- [x] Related issues these changes resolve are linked preferably via [closing keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
- [x] All relevant type-of-change labels added.
- [x] All relevant project fields are set.
- [x] All relevant repo and/or project documentation updated to reflect these changes.
- [x] All relevant functional tests passed.
- [x] All automated checks (e.g., linting, static analysis, unit/smoke tests) passed.

## ✅ Pre-merge checklist ##

- [ ] PR passed smoke test check.
- [ ] Feature branch has been rebased against changes from parent branch, as needed

  Use `Rebase branch` button below or use [this](https://www.digitalocean.com/community/tutorials/how-to-rebase-and-update-a-pull-request) reference to rebase from the command line.
- [ ] Resolved all merge conflicts on branch
- [ ] Notified merge coordinator that PR is ready for merge via comment mention

## ✅ Post-merge checklist ##

- [ ] Feature branch deleted after merge to clean up repository.
- [ ] Verified that all checks pass on parent branch (e.g., main or release-name) after merge.

### CODE REVIEW

File: PowerShell/ScubaGear/baselines/defender.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ The standard and strict preset security policies SHALL be enabled.
 
 
 #### MS.DEFENDER.1.2v1
-All users SHALL be added to Exchange Online Protection in either the standard or strict preset security policy.
+All users SHALL be added to Exchange Online Protection (EOP) in either the standard or strict preset security policy.
```
Comment (gdasher) [inline PowerShell/ScubaGear/baselines/defender.md:89]: is this acronym used in any microsoft documentation?

**Review by gdasher — APPROVED** [review]

File: PowerShell/ScubaGear/baselines/defender.md [hunk 2]
```diff
@@ -483,7 +483,8 @@ Notifications to inform users and help educate them on the proper use of sensiti
 - _Last modified:_ June 2023
 - _Note:_ The custom policy referenced here is the same policy
           configured in [MS.DEFENDER.4.1v1](#msdefender41v1).
-
+- _MITRE ATT&CK TTP Mapping:_
```
Comment (mitchelbaker-cisa) [inline PowerShell/ScubaGear/baselines/defender.md:486]: JCDC missed this change in a few locations: [MS.AAD.3.4](https://github.com/cisagov/ScubaGear/blob/1291-update-iceburg-scbs-to-adjudicate-comments-made-by-jcdc/PowerShell/ScubaGear/baselines/aad.md#msaad34v1), [MS.AAD.6.1](https://github.com/cisagov/ScubaGear/blob/1291-update-iceburg-scbs-to-adjudicate-comments-made-by-jcdc/PowerShell/ScubaGear/baselines/aad.md#msaad61v1), [MS.EXO.5.1](https://github.com/cisagov/ScubaGear/blob/1291-update-iceburg-scbs-to-adjudicate-comments-made-by-jcdc/PowerShell/ScubaGear/baselines/exo.md#msexo51v1), and [MS.EXO.7.1](https://github.com/cisagov/ScubaGear/blob/1291-update-iceburg-scbs-to-adjudicate-comments-made-by-jcdc/PowerShell/ScubaGear/baselines/exo.md#msexo71v1). 

[MS.EXO.10.2](https://github.com/cisagov/ScubaGear/blob/1291-update-iceburg-scbs-to-adjudicate-comments-made-by-jcdc/PowerShell/ScubaGear/baselines/exo.md#msexo102v1) contains the links but not the _MITRE ATT&CK TTP Mapping:_ header.

Screenshot (193)

File: PowerShell/ScubaGear/baselines/exo.md [hunk 3]
```diff
@@ -798,6 +802,7 @@ Emails identified as containing malware SHALL be quarantined or dropped.
 Preventing emails with known malware from reaching user mailboxes helps ensure
 users cannot interact with those emails.
 - _Last modified:_ June 2023
+ - _MITRE ATT&CK TTP Mapping:_
```
Comment (mitchelbaker-cisa) [inline PowerShell/ScubaGear/baselines/exo.md:805]: Small indenting fix

Screenshot (208)

```suggestion
- _MITRE ATT&CK TTP Mapping:_
  - [T1566: Phishing](https://attack.mitre.org/techniques/T1566/)
    - [T1566.001: Spearphishing Attachment](https://attack.mitre.org/techniques/T1566/001/)
```

**Review by mitchelbaker-cisa — APPROVED** [review]

File: PowerShell/ScubaGear/baselines/defender.md (hunk 1 above)
Comment (ahuynhMITRE) [reply PowerShell/ScubaGear/baselines/defender.md:89]: yep, linked below for reference!
https://learn.microsoft.com/en-us/defender-office-365/eop-about

**Comment by ahuynhMITRE** [comment]
@nanda-katikaneni good to merge!

**Pull request merged**